test/unit/RetrievesFinder.test.ts

```diff
@@ -17,6 +17,7 @@ const EXPECTED_VS_RETRIEVE_RESULTS: { results: DataTypeQuery[]; withErrors: Grac
       retrieveLocalId: '14',
       queryLocalId: undefined,
       retrieveLibraryName: 'SimpleQueries',
+      templateId: 'http://hl7.org/fhir/StructureDefinition/Condition',
```

Comment:
Should these templateId's be using QI-Core profiles?
Note that CQL can have multiple "using" lines, e.g., using FHIR version '4.0.1' and using QICore version '4.1.1'
Would that result in multiple templateIds?

Reply:
This is seemingly where they are on the ELMRetrieves, and they seemingly only have one (the type of `templateId` on the `ELMRetrieve` interface in `ELMTypes.ts` is `string | undefined`). Unless you think that ELMRetrieve interface definition needs to be reworked. @hossenlopp maybe you have some more insight on this as well?

Reply:
The `templateId` in the ELM Retrieve may or may not be populated by just a single profile. So this is correct.

The test data here really can be anything, we are just making sure we collect the profile from our scan of the ELM wether it is a base FHIR profile or a QICore one. Nothing is being done here to interpret it.